Makefile

```diff
@@ -98,30 +118,9 @@ integration-test: docker-test
 	# sudo docker run --rm -it quay.io/fossa/fossa-cli-test
 
 # Release tasks.
-.PHONY: prepare-release
-prepare-release:
-	if [ -z "$$RELEASE" ]; then exit 1; fi
-	mv install.sh install.prev.sh
-	make -s installer > install.sh
-	git add install.sh
-	git commit -m "release($(RELEASE)): Release version $(RELEASE)" || make abort-release
-	git tag $(RELEASE) || (git reset HEAD^ && make abort-release)
-	rm install.prev.sh
+install.sh: $(GODOWNLOADER)
+	godownloader --repo=fossas/fossa-cli > install.sh
 
 .PHONY: release
-release:
-	if [ -z "$$(git tag -l --points-at HEAD)" ]; then exit 1; fi
-	[ "$$(grep "^  RELEASE='$$(git tag -l --points-at HEAD)'$$" install.sh | wc -l)" = "1" ]
+release: $(GORELEASER) install.sh
 	GOVERSION=$$(go version) goreleaser $(GORELEASER_FLAGS)
```

Comment:
Is it necessary to separate GORELEASER_FLAGS as a variable when they are only used once?
Use cases I can think of would be to make the makefile easier to modfiy, but I would be curious to hear the reasoning behind this.

Reply:
> make the makefile easier to modify

Yup, that's basically it.